gvsbuild/tools.py

```diff
@@ -106,13 +106,13 @@ def unpack(self):
 @tool_add
 class ToolMeson(Tool):
     def __init__(self):
-        self.version = "0.63.0"
+        self.version = "0.63.1"
```

Comment:
The .2 release is out by now, so I suggest going straight to that.

Reply:
Thanks @eli-schwartz, bumped to latest.